distribution/systemd/vector@.service

```diff
@@ -0,0 +1,23 @@
+[Unit]
```

Comment:
Do we need to keep this in-sync with the `vector.service` file? Just wondering if there is a way we could either templatize this or, at the least, make it clear to people updating one of the files that they need to update the corresponding one.

Reply:
Not that I recall seeing - I believe for example apache provides a single unit and a templateable unit file for the server.